frontend/src/features/collection/state/collectionState.ts

```diff
@@ -24,7 +24,15 @@ export const selectCollection = (state: RootState): VersionInfo[] => {
     }
     ordered.push(collection[id])
   })
-  return ordered
+  return ordered.sort((a, b) => {
+    const aVal = `${a.username}/${a.name}`
+    const bVal = `${b.username}/${b.name}`
+    if (aVal === bVal) {
+      return 0
+    } else {
+      return (aVal < bVal) ? -1 : 1
+    }
```

Comment:
string templating in a sort function is expensive. Instead try this:

```ts
return ordered.sort((a,b) => {
  if (a.username === b.username && a.name === b.name) { return 0 }
  else if (a.username < b.username ||  a.name < b.name) { return -1 }
  return 1
})
```

definitely play with this refactored function to make sure it behaves the same